### PR TITLE
Updated the README file and added a more comprehensive Getting Starte…

### PR DESCRIPTION
**Pull request by tgraupne**
…d Guide

### CODE REVIEW

**Comment by tgraupne** [comment]
🙁 @florianschmidt1994, could you help me out with those eslint checks?

**Comment by florianschmidt1994** [comment]
@tgraupne don't worry about the checks, for now they are still failing for some reason that I still have to debug

File: README.md [hunk 1]
```diff
@@ -37,6 +54,78 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+
+Setup Authentification, the method is `Email and Password`.
+
+Now you can add a firebase App to your project, more precises a `Web App`. Once created, you can edit and download the specific configuration of your app. This configuration contains api keys, backend urls and so on.
+
+Please update the `firebaseConfig.jsx` file in the `src` folder of thise project, to connect your local instance with your specific backend.
+
+### Local Backend Setup
+
+Even though the fire firebase plan is ok for development, it has it's downsides. The following feature which we does not work with the free Spark plan: `scheduled firebase function`.
```
Comment (florianschmidt1994) [inline README.md]: 'free' instead of 'fire' firebase plan?

Comment (florianschmidt1994) [reply README.md]: "The following feature which we does not work with the free Spark plan: `scheduled firebase function`." theres an issue with this sentence

File: README.md [hunk 2]
```diff
@@ -37,6 +54,78 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Backend development
+
+The following steps will explain how to set up your own google firebase project to develop and test cloud functions. Those cloud functions, together with firebase document store `firestore` provide the backend of our project.
+
+DISCLAIMER: The free Spark plan of google firebase is sufficient for development purposes.
+
+First, go to https://firebase.google.com/, take a look at the Geeting Started Guid there and create your own firebase project. At this point, we hope you alread have a google account, because you'll need one.
```
Comment (florianschmidt1994) [inline README.md]: Guid**e**

Comment (florianschmidt1994) [reply README.md]: alread**y**

Comment (timjb) [reply README.md]: Geeting -> Getting

File: README.md [hunk 3]
```diff
@@ -37,6 +54,78 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+
+Once your project is created, you need to configure the following things:
+
+- create a firestore database
+- configure the access rules of your database for proper access
+- create a collection called `stats` with the `document id: "external"`
+
+Setup Authentification, the method is `Email and Password`.
+
+Now you can add a firebase App to your project, more precises a `Web App`. Once created, you can edit and download the specific configuration of your app. This configuration contains api keys, backend urls and so on.
```
Comment (timjb) [inline README.md]: precises -> precisely

File: README.md [hunk 4]
```diff
@@ -37,6 +54,78 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
+
+Please update the `firebaseConfig.jsx` file in the `src` folder of thise project, to connect your local instance with your specific backend.
+
+### Local Backend Setup
+
+Even though the fire firebase plan is ok for development, it has it's downsides. The following feature which we does not work with the free Spark plan: `scheduled firebase function`.
+
+Hence, we need to apply a little change to the code: please rcomment out the creation of:
```
Comment (timjb) [inline README.md]: rcomment -> comment

File: README.md [hunk 5]
```diff
@@ -27,42 +39,74 @@ Your app is ready to be deployed!
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+Once your project is created, you need to configure the following things:
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+- create a firestore database
+- configure the access rules of your database for proper access
+- create a collection called `stats` with the `document id: "external"`
 
-## Learn More
+Setup Authentification, the method is `Email and Password`.
```
Comment (timjb) [inline README.md]: authentification -> authentication

File: README.md [hunk 6]
```diff
@@ -27,42 +39,74 @@ Your app is ready to be deployed!
...
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Now you can add a firebase App to your project, more precises a `Web App`. Once created, you can edit and download the specific configuration of your app. This configuration contains api keys, backend urls and so on.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Please update the `firebaseConfig.jsx` file in the `src` folder of thise project, to connect your local instance with your specific backend.
```
Comment (timjb) [inline README.md]: thise -> this

File: README.md [hunk 7]
```diff
@@ -27,42 +39,74 @@ Your app is ready to be deployed!
...
 
-### Code Splitting
+### Local Backend Setup
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+Even though the free firebase plan is ok for development, it has it's downsides. The following feature does not work with this plan: `scheduled firebase function`.
```
Comment (timjb) [inline README.md]: it's downsides -> its downsides (possessive pronoun)

File: README.md [hunk 8]
```diff
@@ -1,6 +1,23 @@
...
+
+## Getting Started
+
+This repository contains the source code of our website and the following guide explain how to get started and what you need to improve and use this project. Our firebase functions are hosted here: https://github.com/florianschmidt1994/quarantaenehelden-firebase-functions
+
+You need the following libraries to get started: `node`, `yarn`, `firebase cli`. So please install them in a appropriate way on your host system. On macOS, you'd also need the `Xcode command line tools`.
+
+Once you've installed those dependencies run the following inside the project folder
+
+```npm install```
```
Comment (timjb) [inline README.md]: `yarn install`?

File: README.md [hunk 9]
```diff
@@ -1,6 +1,23 @@
...
+
+This will innstall all Javascript dependencies.
```
Comment (timjb) [inline README.md]: innstall -> install

File: README.md (hunk 8 above)
Comment (tgraupne) [reply README.md]: I did not need to do this.
I just assume, that `yarn start` is doing multiple things.

**Comment by florianschmidt1994** [comment]
Merged with cc07bf4

**Pull request closed without merging**